### PR TITLE
Adding wn-oasis.md converted twiki document

### PR DESCRIPTION
**Pull request by djw8605**
https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/UsingOSGWnClientFromOASIS

I also updated the links in the other WN docs.

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Add migration header to TWiki document

### CODE REVIEW

File: docs/client/wn-oasis.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+
+About This Guide
+----------------
+
+The *OSG Worker Node Client* is a collection of software components that is expected to be added to every worker node that can run OSG jobs. It provides a common environment and a minimal set of common tools that all OSG jobs can expect to use. See the [reference section](#Reference) below for contents of the Worker Node Client.
+
+It is possible to install the Worker Node Client software in a variety of ways, depending on what works best for distributing and managing software at your site:
+
+-   Use the Worker Node Client software from OASIS (this guide) - useful when [OASIS](cvmfs.md) is already mounted on your worker nodes
```
Comment (brianhlin) [inline docs/client/wn-oasis.md]: I don't believe the CVMFS doc has been fully converted yet, we should refer to the TWiki doc until then

File: docs/client/wn-oasis.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
+Configuring a Site to Use the Worker Node Client Software From OASIS
+====================================================================
+
+
+About This Guide
+----------------
+
+The *OSG Worker Node Client* is a collection of software components that is expected to be added to every worker node that can run OSG jobs. It provides a common environment and a minimal set of common tools that all OSG jobs can expect to use. See the [reference section](#Reference) below for contents of the Worker Node Client.
+
```
Comment (brianhlin) [inline docs/client/wn-oasis.md]: I think `#Reference` should be `#reference`

File: docs/client/wn-oasis.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+-   MyProxy
+-   osg-system-profiler
+-   osg-version
+-   UberFTP
+-   vo-client (includes /etc/vomses file)
+-   VOMS client
+-   wget
+-   xrdcp
+
+To see the currently installed version of the worker node package, run the following command:
```
Comment (brianhlin) [inline docs/client/wn-oasis.md]: This won't work with the OASIS install, right? I think we should just get rid of this entire section

**Comment by djw8605** [comment]
@brianhlin addressed the comments.

**Pull request merged**